CMR-Metadata-Script/cmrmetadata.py

```diff
@@ -0,0 +1,65 @@
+import csv
+from datetime import datetime
+from datetime import timezone
+from pip._vendor import requests
+import uuid
+from config import BaseConfig
+
+base_url = BaseConfig.CMR_BASE_URL
+query_url = base_url + BaseConfig.GRANULES_URL
+
+filename = BaseConfig.OUTPUT_FILE_NAME
+extension = BaseConfig.OUTPUT_FILE_EXTENSION
+
+outfile = filename + extension
+
+concept_id = BaseConfig.CONCEPT_ID
+
+page_num = BaseConfig.PAGE_NUM
+page_size = BaseConfig.PAGE_SIZE
+
+s3_bucket_name = BaseConfig.S3_BUCKET_NAME
+
+start_date = BaseConfig.START_DATE
+end_date = BaseConfig.END_DATE
+
+date_range = start_date + ',' + end_date
+
+bounding_box = BaseConfig.BOUNDING_BOX
+
+page_num_limit = BaseConfig.PAGE_NUM_LIMIT
+
+def convertToDateTimeFormat(dateTimeStr):
+    return datetime.fromisoformat(dateTimeStr).astimezone(timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
+
+def writeToCsvFile(s3_bucket_name, granule):
+    Id = uuid.uuid1()
+    Name = link["href"].split(s3_bucket_name)[1]
```

Comment:
Maybe do a check to access the index? Could throw error if split doesn't give anything.